R/lbt05.R

```diff
@@ -19,11 +21,14 @@
 lbt05_main <- function(adam_db,
                        arm_var = "ACTARM",
                        lbl_overall = NULL,
+                       map = NULL,
                        ...) {
   assert_all_tablenames(adam_db, c("adsl", "adlb"))
   assert_string(arm_var)
   assert_string(lbl_overall, null.ok = TRUE)
   # expand.grid steps requires levels later.
+  assert_data_frame(map, ncols = 2, null.ok = TRUE)
```

Comment:
what about 0 rows? would that be ok?

Reply:
good point, it makes it bug in the rtables part. Let me reinforce the assertion.